cli/cli.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
```

Comment:
Shouldn't the file extension here be `cli.js`?

Reply:
Good point. Fixed in cc1cf7bfd605100e8606b6b67f1ae5d727ce284a